pkg/config/env/olympus.go

```diff
@@ -1,8 +1,30 @@
 package env
 
-import "github.com/gobuffalo/envy"
+import (
+	"runtime"
+	"strconv"
+
+	"github.com/gobuffalo/envy"
+)
 
 // OlympusGlobalEndpointWithDefault returns Olympus global endpoint defined by OLYMPUS_GLOBAL_ENDPOINT.
 func OlympusGlobalEndpointWithDefault(value string) string {
 	return envy.Get("OLYMPUS_GLOBAL_ENDPOINT", value)
 }
+
+// AthensMaxConcurrency retrieves maximal level of concurrency based on ATHENS_MAX_CONCURRENCY.
+// Defaults to number of cores if env is not set.
+func AthensMaxConcurrency() int {
+	defaultMaxConcurrency := runtime.NumCPU()
+	maxConcurrencyEnv, err := envy.MustGet("ATHENS_MAX_CONCURRENCY")
```

Comment:
here you could use `envy.Get("ATHENS_MAX_CONCURRENCY", defaultMaxConcurrency)` and skip the err check

Reply:
it accepts string, i wanted to avoid these funny conversions

Reply:
ohh, yes my mistake